core-bundle/src/Routing/Enhancer/InputEnhancer.php

```diff
@@ -52,8 +48,8 @@ public function enhance(array $defaults, Request $request): array
         /** @var Input $input */
         $input = $this->framework->getAdapter(Input::class);
 
-        if ($this->prependLocale && !empty($defaults['_locale'])) {
-            $input->setGet('language', $defaults['_locale']);
+        if ('' !== $page->languagePrefix) {
+            $input->setGet('language', $page->rootLanguage);
```

Comment:
Do we even still need that? There has been `$GLOBALS['TL_LANGAUGE']` for ages and nobody should've used `$_GET['language']`, right?

Reply:
Stil removing it would be a BC break. 🤷‍♂️

Reply:
Still removing it would be a BC break. 🤷‍♂️

Reply:
I didn't even know that existed 😁